Mathlib/RingTheory/MvPowerSeries/Basic.lean

```diff
@@ -732,6 +732,16 @@ theorem algebraMap_apply {r : R} :
   change (MvPowerSeries.map σ (algebraMap R A)).comp (C σ R) r = _
   simp
 
+/-- Change of coefficients in mv power series, as an `AlgHom` -/
+def mapAlgHom {σ : Type*} {R : Type*} [CommSemiring R] {S : Type*}
+    [Semiring S] [Algebra R S] {T : Type*} [Semiring T] [Algebra R T]
+    (φ : S →ₐ[R] T) :
+    MvPowerSeries σ S →ₐ[R] MvPowerSeries σ T where
+  toRingHom   := MvPowerSeries.map σ φ
+  commutes' r := by
+    simp only [RingHom.toMonoidHom_eq_coe, OneHom.toFun_eq_coe, MonoidHom.toOneHom_coe,
+      MonoidHom.coe_coe, MvPowerSeries.algebraMap_apply, map_C, RingHom.coe_coe, AlgHom.commutes]
```

Comment:
This is missing some API. Maybe add `@[simps]` or lemmas like `coeff_mapAlgHom` or `mapAlgHom_X`

Reply:
I imagine I could write `mapAlgHom_apply`. What would `@[simps]` do?

Reply:
I was hoping that it would give `mapAlgHom_apply` but I have not tried it myself.

Reply:
It does, but not the one I would wish for (it is already expanded up to the coefficients).